.code-samples.meilisearch.yaml

```diff
@@ -295,15 +465,45 @@ landing_getting_started_1: |-
 
   var task = await client.Index("movies").AddDocumentsAsync<Movie>(documents);
 post_dump_1: |-
+  await client.CreateDumpAsync();
 phrase_search_1: |-
+  await client.Index("movies").SearchAsync<Movie>("'african american' horror");
```

Comment:
For this one we need to use `"` but I'm not sure which one is better this one:
```c#
  await client.Index("movies").SearchAsync<Movie>("\"african american\" horror");
```
or this one:
```c#
  await client.Index("movies").SearchAsync<Movie>('"african american" horror');
```

Reply:
In this case the first one.

Second one won't compile, inside single quote is a char.